modules/virt-importing-vm-datavolume.adoc

```diff
@@ -4,82 +4,82 @@
 [id="virt-importing-vm-datavolume_{context}"]
-= Importing a virtual machine image into a persistent volume claim by using a data volume
+= Importing a virtual machine image into storage by using a data volume
 
-You can import a virtual machine image into a persistent volume claim (PVC) by using a data volume.
+You can import a virtual machine image into storage by using a data volume.
 
-The virtual machine image can be hosted at an HTTP or HTTPS endpoint, or the image can be built into a container disk and stored in a container registry.
+The virtual machine image can be hosted at an HTTP or HTTPS endpoint or the image can be built into a container disk and stored in a container registry.
 
-To create a virtual machine from an imported virtual machine image, specify the image or container disk endpoint in the `VirtualMachine` configuration file before you create the virtual machine.
+You specify the data source for the image in a `VirtualMachine` configuration file. When the virtual machine is created, the data volume with the virtual machine image is imported into storage.
 
 .Prerequisites
 
-* You have installed the OpenShift CLI (`oc`).
-* Your cluster has at least one available persistent volume.
 * To import a virtual machine image you must have the following:
-** A virtual machine disk image in RAW, ISO, or QCOW2 format, optionally
-compressed by using `xz` or `gz`.
-** An HTTP endpoint where the image is hosted, along with any authentication
-credentials needed to access the data source. For example: `http://www.example.com/path/to/data`
-* To import a container disk you must have the following:
-** A container disk built from a virtual machine image stored in your container image registry, along with any authentication credentials needed to access the data source. For example: `docker://registry.example.com/container-image`
+** A virtual machine disk image in RAW, ISO, or QCOW2 format, optionally compressed by using `xz` or `gz`.
+** An HTTP or HTTPS endpoint where the image is hosted, along with any authentication credentials needed to access the data source.
+* To import a container disk, you must have a virtual machine image built into a container disk and stored in a container registry, along with any authentication credentials needed to access the data source.
+* If the virtual machine must communicate with servers that use self-signed certificates or certificates not signed by the system CA bundle, you must create a config map in the same namespace as the data volume.
 
 .Procedure
 
-. Optional: If your data source requires authentication credentials, edit the
-`endpoint-secret.yaml` file, and apply the updated configuration to the cluster:
+. If your data source requires authentication, add the credentials to your cluster:
+
+.. Create a `Secret` manifest, specifying the data source credentials:
 +
 [source,yaml]
 ----
 apiVersion: v1
 kind: Secret
 metadata:
-  name: <endpoint-secret>
+  name: <endpoint-secret> <1>
```

Comment:
If the name of the secret in the example is "endpoint-secret" you can remove the brackets (< >). If it's meant to be a generic term, please change the hyphen to an underscore